composer.json

```diff
@@ -39,14 +39,14 @@
     "require-dev": {
         "ext-sqlite3": "*",
         "symfony/browser-kit": "^5.4|^6.0|^7.0",
-        "doctrine/doctrine-bundle": "^1.12|^2.0",
```

Comment:
One request here, can you check so we don't bump this to the latest version but instead try to use the minimal version that would work (the opposite of how you would do for an application using this) - to allow people an easier upgrade path?

Reply:
This is in the require-dev section. So i guess this should not make the upgrade path harder. A new major version for this PR is planned anyway.

Reply:
https://packagist.org/packages/doctrine/doctrine-bundle#2.11.0 adds the support for symfony 7

Reply:
Ah, true - thanks!